frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -182,44 +215,106 @@ class DatasetImportView extends React.PureComponent<Props, State> {
       });
     }
 
+    let message;
+    let type = "info";
+    const didParsingTheSavedDataSourceJSONSucceed = savedDataSourceOnServer.status == null;
+    if (!didParsingTheSavedDataSourceJSONSucceed) {
+      message = (
+        <div>
+          The current datasource-properties.json on the server seems to be invalid and webKnossos
+          could not parse this file. The settings below are suggested by webKnossos and should be
+          adjusted. <br />
+          Be aware that webKnossos cannot guess properties like the voxel size or the largest
+          segment id. You must set them yourself.
+        </div>
+      );
+      type = "warning";
+    } else if (!isDatasourceJSONValid(savedDataSourceOnServer)) {
+      // TODO: why it this never the case? Shall this case be removed?
+      message = (
+        <div>
+          The current datasource-properties.json on the server seems to be invalid. Thus the
+          datasource-properties.json settings below were suggested by webKnossos. The current
+          content of the datasource-properties.json can be viewed{" "}
+          <a
```

Comment:
I got a question regarding this case. @fm3 I think you can answer my question here.

This case happens/should happen when the datasource-properties.json is in a valid JSON Format but the JSON does not match the datasource-properties.json schema. One reason might be that a properties is missing or so.
To reproduce the intended error to catch with this case, simply remove the "largestSegmentId" from the JSON.

While working at this PR, I noticed that the backend also validated the format and if that fails, simply sends an error and not the invalid JSON :/
The message the backend sends to the frontend when the largestSegmentId property is missing is:
```
status: "Error: Invalid json format in binaryData/sample_organization/e2006_knossos/datasource-properties.json: Failure(Invalid json: List((/dataLayers(1)/dataFormat/category/largestSegmentId,List(JsonValidationError(List(error.path.missing),WrappedArray())))),Empty,Empty)"
```

@fm3 Is it possible that the backend still sends the invalid JSON? This would be helpful as then the only invalid option is largestSegmentId and the rest of the set options could be used to fill the settings view. This way the user does only need to set the largestSegmentId and not the other fields like the bounding boxes or so.

Reply:
I would have expected that the values »suggested by webknossos« as sent by the backend do contain the values of those fields that were valid in the original? I think some merging goes on there in the backend already.

Reply:
if that is not the case or that does not support the optimal workflow, sure, we can change the backend behavior, do you think that would be beneficial here?

Reply:
I just check this and the backend does not merge the bounding boxes, scale, voxel size and largest segment id. 

The use case would be that repairing a broken datasource-properties.json would be easier as only the missing / broken attribute has to be adjusted and not very of the attribute listed above.

I think what is against this case is whether this really happens or if a broken datasource-properties.json is very rare. 
I think: if it is rare then it should not be necessary to add this to the backend but if this happens from time to time I think that would make fixing the JSON file easier.

Reply:
@MichaelBuessemeyer and I just had a discussion. We agreed on the following approach:
If the backend can not validate the `datasource_properties.json`, it creates an `UnusableDatasource` as before, but checks if the `datasource_properties.json` is valid json. If it is valid json, it is saved alongside the current status.
When exploring an `UnusableDatasource`, the `previousDatasource` response field is set to the valid, but unusable json, or to the normal `UnusableDatasource` if the json is invalid.

Reply:
Thanks, @youri-k :smile: 
@fm3 just for your notice:
Youri's changes made case three of the listed ones above possible. Without those changes, case three would be exactly handled as case 4.